docs/getting-started/testnet-azure-1click.md

```diff
@@ -0,0 +1,224 @@
+
+```bash
+Generated private validator file             module=main keyFile=/chain/.chain-maind/config/priv_validator_key.json stateFile=/chain/.chain-maind/data/priv_validator_state.json
+🔄 Regenerate tmkms consensus-ed25519 key 🔑
+   Generated consensus (Ed25519) private key at: /chain/.tmkms/secrets/consensus-ed25519.key
+🔄 Regenerate tmkms validator secret key 🔑
+🕑 Waiting for tmkms to run
+
+✅ Please keep consensus public key for node join if it is validator or find it again in /chain/log/tmkms/tmkms.log. It will show again when restart tmkms in log
+
+Consensus public key for node join: tcrocnclconspubXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+```
+
+The script will also ask you to fill in the `moniker` value which is a human readable display name for tendermint p2p.
+
+```bash
+Replace moniker in /chain/.tendermint/config/config.toml
+Moniker is display name for tendermint p2p
+
+moniker: YOUR_MONIKER_NAME
+```
+
+You may also enable [STATE-SYNC](https://docs.tendermint.com/master/tendermint-core/state-sync.html) to speed up the block syncing process.
+_p2p gossip_ will allow you to connect with more nodes (data sources).
+```bash
+Do you want to enable state-sync? (Y/N): y
+Do you want to add the public IP of this node for p2p gossip? (Y/N): y
+✅ Added public IP to external_address in chain-maind config.toml for p2p gossip
+```
+_node_id_ and _node_key_ are the unique identity of your node. Save them for later use.
+
+```
+Generate and replace node_key in /chain/.chain-maind/config/node_key.json
+
+You may want to save node_id and node_key for later use
+
+node_id: 62cf74XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+node_key: LyQiGlL4HsdHsPXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+```
+`chain-maind` is now running at the background. 
+
+You can check its log by the command `journalctl`, for example:
+
+```bash
+$ journalctl -u chain-maind.service -f
+-- Logs begin at Mon 2020-11-23 13:21:03 UTC. --
+Nov 25 09:27:41 crypto-chain-test-node chain-maind[3209]: I[2020-11-25|09:27:41.854] Executed block                               module=state height=545512 validTxs=0 invalidTxs=0
+Nov 25 09:27:41 crypto-chain-test-node chain-maind[3209]: I[2020-11-25|09:27:41.866] Committed state                              module=state height=545512 txs=0 appHash=B989A1B3E225A6BCBB4D61072E83A9EB3F9C480CE8CFDBCC4B3469620D314F99
+Nov 25 09:27:48 crypto-chain-test-node chain-maind[3209]: I[2020-11-25|09:27:48.271] Executed block                               module=state height=545513 validTxs=1 invalidTxs=0
+Nov 25 09:27:48 crypto-chain-test-node chain-maind[3209]: I[2020-11-25|09:27:48.287] Committed state                              module=state height=545513 txs=1 appHash=D55A839445CD58DD8EAD27D8FA87ACAF88695BE47CDE4FFCD456421EB7800775
+...
+```
+
+Once the tendermint syncs to the latest block, the setup is done! You may move on to the next step if you want to join the network as a validator.
```

Comment:
The latest block height can be found by
`curl -s https://testnet-croeseid-1.crypto.com:26657/block | jq -r .result.block.header.height`